src/parallel/bbssrv.h

```diff
@@ -17,5 +17,8 @@
 #define LOOK_TAKE_RESULT 14
 #define TAKE_TODO        15
 #define CONTEXT          16
-#define FIRSTID          17
+#define CRAY_POST        17
+#define CRAY_POST_TODO   18
+#define CRAY_POST_RESULT 19
```

Comment:
@alkino : should we remove `CRAY_` prefix from these?

Reply:
I don't think so, the only problem is that we cannot move numbers but they are not use